partial revert to 1.47 in order to work with an old perl 5.10.1

With old ExtUtils::MakeMaker, AUTHOR takes a string/number not a ARRAY reference.
See http://api.metacpan.org/source/RUZ/GDGraph-1.48/CHANGES

Signed-off-by: Francois Perrad <francois.perrad@gadz.org>

Index: b/Makefile.PL
===================================================================
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -19,11 +19,7 @@
 
         ($] >= 5.005 ?
             ('ABSTRACT'	    => 'Produces charts with GD',
-             'AUTHOR'       => [
-                 'Martien Verbruggen <mgjv@tradingpost.com.au>',
-                 'Benjamin Warfield <bwarfield@cpan.org>',
-                 'Ruslan Zakirov <Ruslan.Zakirov@gmail.com>',
-            ]
+             'AUTHOR'	    => 'Martien Verbruggen <mgjv@tradingpost.com.au>',
             ):()
         ),
 
